acct-user/jackett/jackett-0-r1.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-user
+
+DESCRIPTION="User for jackett"
+ACCT_USER_HOME=/var/lib/jackett
+ACCT_USER_ID=531
```

Comment:
Please use -1 when submitting a PR, then the merger can allocate one.

Reply:
Sorry, I didn't know that.

Reply:
I took care of making sure the UID/GID was available and reserving it.

But in the future, `-1` is the suggested approach.

Thanks again!